src/Console/Command/CollectMetricsCommand.php

```diff
@@ -60,5 +60,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         } else {
             $this->aggregateMetricsCron->execute();
         }
+
+        return Command::SUCCESS;
```

Comment:
Could you please use `\Magento\Framework\Console\Cli::RETURN_SUCCESS` here?

Reply:
Done!